db/schema.rb

```diff
@@ -160,8 +160,6 @@
     t.string "key", null: false
     t.string "value", null: false
     t.boolean "is_translatable", default: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
```

Comment:
Was this intentional? I don't think these should be deleted unless it was intentional and there's a reason for it.